Libs/MRML/Core/vtkMRMLSegmentationStorageNode.cxx

```diff
@@ -923,6 +923,11 @@ int vtkMRMLSegmentationStorageNode::ReadPolyDataRepresentation(vtkMRMLSegmentati
     {
     // Get poly data representation
     vtkPolyData* currentPolyData = vtkPolyData::SafeDownCast(multiBlockDataset->GetBlock(blockIndex));
+    if (!currentPolyData)
+      {
+      vtkErrorMacro("ReadPolyDataRepresentation: Could not read block " << blockIndex);
+      continue;
```

Comment:
Can it cause any issue elsewhere that the polydata is not set? We could set an empty polydata object in the representation instead.

If we create a new empty segment, is it initialized with an empty polydata or no representation is added at all?

Reply:
In this case we would avoid creating the segment entirely (since we don't know the name or id), so there wouldn't be any issues if the polydata wasn't set.

When we create a new segment, an empty polydata is set as the representation.